build/media_source/com_templates/scss/pagebuilder.scss

```diff
@@ -0,0 +1,136 @@
+// App.vue
+.settings {
+  z-index: 1040;
+  padding: .5rem;
+}
+
+fieldset {
+  .btn-group {
+    button.btn-danger {
+      margin-left: 0.75rem!important;
+    }
+  }
+}
+
+button, input, select, textarea {
+  border-style: solid;
+  background-color: #ffffff;
+  color: #000000;
+}
+
+select {
+  -webkit-appearance: menulist;
+}
+
+.pagebuilder {
+  // Min-height needed for left drawer
+  min-height: 30em;
+
+  .row-wrapper {
+    position: relative;
+    border: 1px dashed #4b4b4b;
+    padding: 2rem 0;
+    min-height: 15rem;
+    // Repeating background needed to overwrite Vuetify
+    background: linear-gradient(to right, rgba(0, 0, 0, .1) 1px, transparent 1px), linear-gradient(rgba(0, 0, 0, .1) 1px, transparent 1px) repeat;
+    // background-size needed for easy modifying by js (default 12 columns)
+    background-size: 8.3% 150px;
+
+    &:hover {
+      background-color: #ececec;
+    }
+
+    button {
+      cursor: pointer;
+
+      &:hover {
+        cursor: pointer;
+      }
+    }
+
+    .btn-add {
+      display: block;
+      margin: 1rem auto 0;
+      min-width: 6rem;
+      width: 30%;
+
+      &:not(:hover) {
+        background-color: #fff;
+      }
+    }
+
+    .btn-wrapper {
+      visibility: hidden;
+      position: absolute;
```

Comment:
```
    visibility: hidden;
    position: absolute;
    top: -32px;
    right: 0;
    background-color: #9898ea;
}
```

Reply:
If you make this change this will looks something like 
![Screenshot from 2019-06-28 22-03-53](https://user-images.githubusercontent.com/30978328/60357242-c05c8380-99f0-11e9-9934-8f8f9368aa5f.png)

Reply:
If its looks good then  You can change the background color you want.

Reply:
Done. Thanks for the draft!